workspaces/frontend/src/app/pages/Workspaces/Workspaces.tsx

```diff
@@ -579,82 +599,90 @@ export const Workspaces: React.FunctionComponent = () => {
+                            )
+                          : getRedirectStatusIcon(undefined, 'No API response available')}
+                      </Td>
+                      <Td dataLabel={columnNames.name}>{workspace.name}</Td>
+                      <Td dataLabel={columnNames.kind}>
+                        {kindLogoDict[workspace.kind] ? (
+                          <Tooltip content={workspace.kind}>
+                            <Brand
+                              src={kindLogoDict[workspace.kind]}
+                              alt={workspace.kind}
+                              style={{ width: '20px', height: '20px', cursor: 'pointer' }}
+                            />
+                          </Tooltip>
+                        ) : (
+                          <Tooltip content={workspace.kind}>
+                            <CodeIcon />
+                          </Tooltip>
+                        )}
+                      </Td>
+                      <Td dataLabel={columnNames.image}>{workspace.options.imageConfig}</Td>
+                      <Td dataLabel={columnNames.podConfig}>{workspace.options.podConfig}</Td>
+                      <Td dataLabel={columnNames.state}>
+                        <Label color={stateColors[workspace.status.state]}>
+                          {WorkspaceState[workspace.status.state]}
+                        </Label>
+                      </Td>
+                      <Td dataLabel={columnNames.homeVol}>{workspace.podTemplate.volumes.home}</Td>
+                      <Td dataLabel={columnNames.cpu}>{`${workspace.cpu}%`}</Td>
+                      <Td dataLabel={columnNames.ram}>{formatRam(workspace.ram)}</Td>
+                      <Td dataLabel={columnNames.lastActivity}>
+                        <Timestamp
+                          date={new Date(workspace.status.activity.lastActivity)}
+                          tooltip={{ variant: TimestampTooltipVariant.default }}
+                        >
+                          1 hour ago
+                        </Timestamp>
+                      </Td>
+                      <Td>
+                        <WorkspaceConnectAction workspace={workspace} />
+                      </Td>
+                      <Td isActionCell data-testid="action-column">
+                        <ActionsColumn
+                          items={workspaceDefaultActions(workspace).map((action) => ({
+                            ...action,
+                            'data-testid': `action-${action.id || ''}`,
+                          }))}
+                        />
+                      </Td>
+                    </Tr>
+                    {isWorkspaceExpanded(workspace) && (
+                      <ExpandedWorkspaceRow workspace={workspace} columnNames={columnNames} />
+                    )}
+                  </Tbody>
+                ))}
+              {sortedWorkspaces.length === 0 && (
+                <Tr>
+                  <Td colSpan={11} id="empty-state">
```

Comment:
```suggestion
                  <Td colSpan={12} id="empty-state">
```

The table has 12 columns including the `columnNames` and the two actions columns.